platform/darwin/src/MGLMapSnapshotter.h

```diff
@@ -119,6 +119,18 @@ MGL_EXPORT
  */
 @property (nonatomic) CGFloat scale;
 
+/**
+ :nodoc:
+ Whether the Mapbox logo is displayed.
```

Comment:
[Per our documentation](https://docs.mapbox.com/help/how-mapbox-works/attribution/), Mapbox attribution contains the Mapbox _wordmark_ and text _attribution_. To be consistent, you might want to consider changing this (and the property name) to "Mapbox wordmark".

Reply:
> It works, but stumbled getting it up and running with Swift, where I ended up needing to make an Objective-C class that overrides showsLogo to get it to work.

Yep, that's no good. Let me rethink.

Reply:
Done - I've kept the properties as `logo` for consistency with `MGLMapView` and Android.